rubocop.yml

```diff
@@ -726,19 +724,18 @@ Metrics/BlockNesting:
   Max: 3
 
 Metrics/ClassLength:
-  CountComments: false  # count full line comments?
-  Max: 100
+  Enabled: false
 
 Metrics/ModuleLength:
-  CountComments: false  # count full line comments?
+  CountComments: false # count full line comments?
   Max: 100
 
 # Avoid complex methods.
 Metrics/CyclomaticComplexity:
   Max: 6
 
 Metrics/LineLength:
-  Max: 100
+  Max: 110
```

Comment:
I think I set this back a while ago because Gitlab shows only 100 characters in the diff. If you have more than that the Diff gets torn apart or you have to scroll. Please check that first.